src/pages/SetUpGradleDocumentation/SetUpAgentGradleDocumentation.tsx

```diff
@@ -62,6 +63,35 @@ const SetupAgentGradleDocumentation: React.FC = () => {
                 marginRight: "auto",
               }}
             />
+          </ul>
+          <strong>Grovy</strong>
+          <ul>
+            <img
+              src="src/assets/images/gradle/Step-01-1 Gradle.png"
```

Comment:
import the images, do not put the path as string
Example:
import image from "../../assets/images/welcome-background.png"
src={image}